apps/api/src/bulk-uploads/raw-data-mapper/mappers/strapi-song-mapper.ts

```diff
@@ -0,0 +1,58 @@
+import { AddSong } from '../../../domain/models/song/commands/add-song.command';
+import { DTO } from '../../../types/DTO';
+import { FieldCalculationRules, RawDataMapping } from '../raw-data-mapping';
+
+type StrapiMedia = {
+    id: number;
+    name: string;
+    hash: string;
+    ext: string;
+    mime: string;
+    size: number;
+    url: string;
+    created_at: string;
```

Comment:
Not that we necessarily need to bring this record data in from strapi, but is this something we'll want in our system?  Or will the command/event history take care of this?